doc/dev/GETTING_STARTED.md

```diff
@@ -81,59 +81,89 @@ Note: The following steps were tested with Visual Studio Community 2022 using Wi
 - Download and install [cmake][cmake-download]
 - Download and install [Visual Studio 2022 Community Edition][visual-studio]
 
-#### Recover F3D sources
+#### Retrieve VTK and F3D sources
 
 - Open git bash
 
 ```sh
 cd C:
 mkdir dev
 cd dev
+mkdir vtk
+cd vtk
+git clone https://gitlab.kitware.com/vtk/vtk.git src
+cd src
+git checkout v9.4.1
+cd ..
+mkdir build
+mkdir install
+cd ..
 mkdir f3d
 cd f3d
 git clone https://github.com/f3d-app/f3d.git src
 mkdir build
 ```
 
-#### Install dependencies using vcpkg
-
-- Open git bash
+Note: Resulting folder architecture should look like this:
 
-```sh
-cd C:/dev
-git clone https://github.com/Microsoft/vcpkg.git
+```
+dev
+|-- f3d
+|   |-- build
+|   `-- src
+`-- vtk
+    |-- build
+    |-- install
+    `-- src
 ```
 
-- Open cmd
+#### Build and install VTK in a dedicated folder
+
+- Open x64 Native Tools Command Prompt for VS 2022
 
 ```sh
-cd C:\dev
-.\vcpkg\bootstrap-vcpkg.bat
-cmake -B .\f3d\build -S .\f3d\src -DCMAKE_TOOLCHAIN_FILE=C:\dev\vcpkg\scripts\buildsystems\vcpkg.cmake
+cd C:\dev\vtk\build
+cmake -DCMAKE_INSTALL_PREFIX=../install ../src
+cmake --build . -j 16 --config Debug
+cmake --install . --config Debug
```

Comment:
I find that cleaner but feel free to ignore if you disagree.
```suggestion
cd C:\dev\vtk
cmake -S src -B build
cmake --build build -j 16 --config Debug
cmake --install build --config Debug --prefix install
```

Reply:
I think this approach is less clear for cmake beginnners.